docs/data_prep.md

```diff
@@ -17,18 +17,39 @@ please prepare the tabular data for your dataset as a single, tab-separated file
 
 ### All datasets
 
-A valid dataset for Neurobagel **must** include a TSV file that describes participant attributes. 
-The TSV must contain a minimum of two columns: at least one column must contain subject IDs, 
-and at least one column must describe demographic or other phenotypic information 
-(for variables currently modeled by Neurobagel, see the [data dictionary section](dictionaries.md)).
+A valid dataset for Neurobagel **MUST** include a TSV file that describes participant attributes.
+
+The TSV: 
```

Comment:
The issue is that there is an order to the requirements. I.e. I first have to say "you may have a session" for the "your session must have unique values" and so on. So in practice it'd look more like 

```
MUST
- ...
- ...

MAY
- ...

MUST
- ...
- ...
```
Not sure that fits nicely here

Reply:
@alyssadai I'll leave this comment open for you to take a look. If you want to suggest a version that does the keyword section split and is more readable, I'll add it. Otherwise I'd say let's merge now and make it better later.

Reply:
I think there is a way to phrase it that still allows keeping all the MUST / MAY statements together. E.g., for the ID columns bullets, I would just suggest:

```
The TSV MUST contain:
- A minimum of two columns
- At least one column containing subject IDs
- At least one column that describes demographic or other phenotypic information
- Unique values in the subject ID column or unique combinations of IDs if both subject and session ID columns are present

The TSV MAY contain:
- A column with session IDs, e.g. if the dataset is longitudinal

The TSV MUST NOT contain:
- Missing values in the subject ID and session ID (if available) columns
```

or something like that. I don't think it's necessary to have the MAY interspersed among the MUST